WpfAnimatedGif/ImageAnimationController.cs

```diff
@@ -60,6 +60,19 @@ public int FrameCount
         }
 
         /// <summary>
+        /// Returns the number of frames in the image.
+        /// </summary>
+        public TimeSpan Duration
+        {
+            get
+            {
+                return _animation.Duration != null && _animation.Duration.HasTimeSpan
+                  ? _animation.Duration.TimeSpan
+                  : TimeSpan.Zero;
```

Comment:
`ImageBehavior.GetAnimation` always sets the `Duration`, so `HasTimeSpan` should always be true, but 👍 for playing it safe ;)